revolv/project/migrations/0050_auto_20150615_0435.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import ckeditor.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('project', '0049_merge'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='MonthlyOrganizationalCost',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('year', models.IntegerField()),
+                ('amount', models.DecimalField(max_digits=8, decimal_places=3)),
+                ('project', models.ForeignKey(to='project.Project')),
+            ],
+            options={
+            },
+            bases=(models.Model,),
+        ),
+        migrations.CreateModel(
+            name='MonthlyReinvestableAmount',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('year', models.IntegerField()),
+                ('amount', models.DecimalField(max_digits=8, decimal_places=3)),
+                ('project', models.ForeignKey(to='project.Project')),
+            ],
+            options={
+            },
+            bases=(models.Model,),
+        ),
+        migrations.AlterField(
+            model_name='donationlevel',
+            name='amount',
+            field=models.DecimalField(max_digits=15, decimal_places=2),
+            preserve_default=True,
+        ),
+        migrations.AlterField(
```

Comment:
I think the duplicate is in [https://github.com/calblueprint/revolv/blob/master/revolv/project/migrations/0049_auto_20150615_0053.py](https://github.com/calblueprint/revolv/blob/master/revolv/project/migrations/0049_auto_20150615_0053.py)